code/game/objects/items/weaponry.dm

```diff
@@ -901,3 +901,15 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	force = 14
 	breakforce = 25
 	stamforce = 5
+
+/obj/item/melee/transforming/hydraulic_blade
+	name = "hydraulic blade"
+	desc = "Extremely dangerous and sharp weapon."
+	icon_state = "hydraulic_blade"
+	item_state = "hydraulic_blade"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	force = 20

```

Comment:
due to autodoc bullshit, nodrop, undisarmability, etc, it's best this be at 18 force, same as a spear.  give it a block_upgrade_walk 1, and block_flags = BLOCKING_ACTIVE | BLOCKING_NASTY

Reply:
done